apps/financial-aid/web-osk/pages/umsokn/hjuskaparstada.tsx

```diff
@@ -0,0 +1,3 @@
+import RelationshipstatusForm from '@island.is/financial-aid-web/osk/src/routes/application/RelationshipstatusForm/RelationshipstatusForm'
+
+export default RelationshipstatusForm
```

Comment:
Isn't Relationship and status two words? Should thei be RelationshipStatusForm ?